package.json

```diff
@@ -46,13 +46,16 @@
   "dependencies": {
     "@babel/runtime": "^7.1.2",
     "@blueprintjs/icons": "^3.2.0",
+    "@segment/consent-manager": "^4.2.2",
+    "@segment/in-regions": "^1.2.0",
     "@types/react": "^16.9.5",
     "arrify": "^1.0.1",
     "classnames": "^2.2.6",
     "dom-helpers": "^3.2.1",
     "downshift": "^3.3.1",
     "fuzzaldrin-plus": "^0.6.0",
     "glamor": "^2.20.40",
+    "lodash": "^4.17.11",
```

Comment:
What's the reason for these package.json changes?

Reply:
Good q..I'll remove this.